### PR TITLE
Changelog

### PR DESCRIPTION
**Pull request by vahurtad**
https://trello.com/c/FNXfBtj1/34-whats-new-change-log

Using https://twitter.com/i/release_notes as inspiration for the changelog scrolling.
Not using special css but bootstrap which works even better.

image
whats-new-scroll-3

### CODE REVIEW

File: src/components/new-feature.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import ArrowRightAlt from '@material-ui/icons/ArrowRightAlt';
+
+
+const NewFeature = (props) => {
+
+    return (
+        <div className="mt-4">
+            <h3>{props.heading}</h3>
+            <div>{props.date}</div>
+            <img src={props.image} className="mw-100 pt-3"/>
```
Comment (jbaldo) [inline src/components/new-feature.js:14]: Problem from my code... there is a warning that we should have `alt` text for accessibility. My plan was to have this content coming from the Contentful CMS in the next phase. Could we just comment out the image for now and we can handle images then?

File: src/pages/whats-new.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+import "../styles/pages/whats-new.scss"
+
+const dateToDivID = (date) => {return "#" + date.replace(/[\W_]+/g, '-').toLowerCase()}
+
+const WhatsNew = () => (
+  <Layout pageClassName="whats-new-page">
+    <SEO title="What's New" />
+    <Container>
+    {/*<Row><Col><h2>What's New</h2></Col></Row>*/}
```
Comment (jbaldo) [inline src/pages/whats-new.js:21]: If this looks OK, could we put it in to make it obvious what the content is? Or if you can think of a better way...

File: src/pages/whats-new.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+                        <ListGroup.Item action >
+                            <Link
+                            activeClass="active"
+                            spy={true}
+                            smooth={true}
+                            offset={-150}
+                            duration= {550} 
+                            to={dateToDivID(feature.date)}>
+                                {feature.date}
+                            </Link>
```
Comment (jbaldo) [inline src/pages/whats-new.js:36]: Took me a bit of playing with it to figure out that you have to click right on the date for it to scroll. Does it still work if you wrap the `ListGroup.item` in the `Link`?

File: src/pages/whats-new.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+const dateToDivID = (date) => {return "#" + date.replace(/[\W_]+/g, '-').toLowerCase()}
+
+const WhatsNew = () => (
+  <Layout pageClassName="whats-new-page">
+    <SEO title="What's New" />
+    <Container>
+    {/*<Row><Col><h2>What's New</h2></Col></Row>*/}
+        <Row className="d-flex bd-highlight">
+            <Col md={4} className="d-none d-sm-block p-2 w-100 bd-highlight" id="left-dates">
+                <ListGroup variant="flush" className="mt-3">
```
Comment (jbaldo) [inline src/pages/whats-new.js:24]: Is the List meant to stick on the left side? or did we scrap that?

File: package.json [hunk 5]
```diff
@@ -30,6 +30,7 @@
     "react-bootstrap-table2-toolkit": "^2.1.0",
     "react-dom": "^16.10.2",
     "react-helmet": "^5.2.1",
+    "react-scroll": "^1.7.16",
```
Comment (jbaldo) [inline package.json:33]: Cool that both your widget and Emily's are using `react-scroll`!

File: src/pages/whats-new.js [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+import NewFeature from "../components/new-feature"
+import Layout from "../components/layout"
+import SEO from "../components/seo"
+
+import newFeaturesData from "../../data/new-features.json"
+import image from "../images/feature-june.gif"
+
+import "../styles/pages/whats-new.scss"
+
+const dateToDivID = (date) => {return "#" + date.replace(/[\W_]+/g, '-').toLowerCase()}
```
Comment (jbaldo) [inline src/pages/whats-new.js:15]: is this basically formatting the date into an id that `react scroll can reference`?

File: src/pages/whats-new.js (hunk 2 above)
Comment (vahurtad) [reply src/pages/whats-new.js:21]: Hi John, is putting the title back be better?

File: src/pages/whats-new.js (hunk 6 above)
Comment (vahurtad) [reply src/pages/whats-new.js:15]: yes!

File: src/pages/whats-new.js (hunk 4 above)
Comment (vahurtad) [reply src/pages/whats-new.js:24]: Right now, the dates show up and are aligned on the left side. Should the dates be aligned towards the right side instead?

File: src/pages/whats-new.js (hunk 3 above)
Comment (vahurtad) [reply src/pages/whats-new.js]: fixed in the latest commit!

**Pull request merged**